assets/css/style.css

```diff
@@ -708,10 +708,10 @@
 
 	#banner {
 		background-image: url("../images/hyrax_logo_horizontal_white_background.png");
-		background-position: center center;
-		background-size: contain;
-                background-repeat: no-repeat;
-		height: 28em;
+		background-position: 50% 10px;
+		background-size: auto 140px;
+    background-repeat: no-repeat;
```

Comment:
Works for me:
<img width="569" alt="screen shot 2016-12-14 at 11 00 21 am" src="https://cloud.githubusercontent.com/assets/92044/21192001/9fd5021a-c1ec-11e6-87a6-e8d08d3368b0.png">

I think this is because the templates you have used tabs rather than spaces.